bigbluebutton/bigbluebutton
# Fix: [3.0] `userdata-bbb_hide_presentation_on_join` parameter is hiding presentation for presenters

### What does this PR do?

This PR fixes the custom parameters `userdata-bbb_hide_presentation_on_join` that was also hiding presentation for presenters.

### Closes Issue(s)

Closes #21322 

### How to test

Start a meeting, apply custom parameter `userdata-bbb_hide_presentation_on_join` , and join with: `Presenter `- `Moderator` and `Viewer` - It should NOT apply only for the `Presenter`.


### More

https://github.com/user-attachments/assets/9e4f4010-38f6-4be2-a6fb-aa94051bb88e
